portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase

```diff
@@ -198,6 +263,521 @@ definition {
+			JSONAssetlist.addDynamicAssetListEntryWithItemType(
+				groupName = "Test Site Name",
+				itemSubtype = "All Subtypes",
+				itemType = "Blogs Entry",
+				title = "Dynamic Collection Test");
+		}
+
+		task ("Filter blogs entry by category") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+
+			AssetListsAdmin.addCategoriesFilter(
+				categoryNameList = "Audience 1,Stage 1",
+				queryAndOperator = "All",
+				queryContains = "Contains");
+		}
+
+		task ("Add a Collection Display to page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name",
+				type = "content");
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addFragment(
+				collectionName = "Content Display",
+				fragmentName = "Collection Display");
+		}
+
+		task ("Select the dynamic collection in the Collection Display") {
+			PageEditor.editCollectionDisplay(
+				assetListName = "Dynamic Collection Test",
+				fragmentName = "Collection Display",
+				styleDisplay = "Bulleted List");
+
+			Button.clickPublish();
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/test-page-name");
+		}
+
+		task ("Assert the correct asset is shown") {
+			AssertTextEquals.assertPartialText(
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_BULLET_LIST_TEXT",
+				value1 = "Blogs Entry");
+		}
+	}
+
+	@description = "LPS-148882: Given a Blog Entry with Audience and/or Stage categories
+	When the user filters by clicking at category's bar
+	Then the Blogs Entry is listed"
+	@priority = "4"
+	test CheckBlogFilterByCategoryBar {
+		property portal.acceptance = "true";
+
+		task ("Add a global category") {
+			var categoryId1 = JSONCategory.addCategory(
+				groupName = "Global",
+				title = "Audience 1",
+				vocabularyName = "Audience");
+			var categoryId2 = JSONCategory.addCategory(
+				groupName = "Global",
+				title = "Stage 1",
+				vocabularyName = "Stage");
+		}
+
+		task ("Create a blog") {
+			HeadlessBlog.addBlogEntry(
+				assetCategoryIds = "${categoryId1},${categoryId2}",
+				content = "Blogs Entry Content",
+				groupName = "Test Site Name",
+				title = "Blogs Entry Title");
+		}
+
+		task ("Go to Content Dashboard and check the asset is being counted at graph") {
+			ApplicationsMenu.gotoSite(site = "Test Site Name");
+
+			ContentDashboard.goToContentDashboard();
+
+			Click(locator1 = "ContentDashboard#COLUMN_FILTER");
+
+			WaitForPageLoad();
+
+			AssertTextPresent(
+				locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED",
+				value1 = "Blogs Entry Title");
+		}
+	}
+
+	@description = "LPS-148882: Given there are two or more sites created
+	When user is creating a Blog Entry
+	Then s/he can add Audience and Stage categories across all sites"
+	@priority = "4"
+	test CheckBlogsCategoriesMoreThanOneSite {
```

Comment:
It's a good idea to add the `@uitest` to this case. As it adds categories using the UI, someone in the future can think is not necessary and add it using JSON (which is not the goal of the test).